x-pack/plugins/apm/server/lib/services/get_throughput.ts

```diff
@@ -27,12 +27,17 @@ interface Options {
 
 type ESResponse = PromiseReturnType<typeof fetcher>;
 
-function transform(response: ESResponse) {
+function transform(response: ESResponse, options: Options) {
+  const { end, start } = options.setup;
+  const deltaAsMinutes = (end - start) / 1000 / 60;
```

Comment:
I see us doing this calculation everywhere we calculate throughput. Can we make a helper? That should make it super obvious when we are not calculating it correctly

Reply:
I was going to do this in another PR, but i can do it now.

Reply:
Actually, that's probably better. This simple fix should go into 7.11, and the helper 7.12

Reply:
Oh, I see you already did it. That's fine too! :D 
Can you verify if we have API test coverage everywhere we calculate throughput? Easiest is probably to remove the call to the helper, and see if an api test fails.